vllm/inputs/registry.py

```diff
@@ -132,33 +135,29 @@ def get_hf_processor(
     def call_hf_processor(
         self,
         hf_processor: ProcessorMixin,
-        prompt: str,
-        processor_data: Mapping[str, object],
-        inference_kwargs: Mapping[str, object],
+        data: Mapping[str, object],
+        kwargs: Optional[Mapping[str, object]] = None,
     ) -> BatchFeature:
         assert callable(hf_processor)
 
+        if kwargs is None:
+            kwargs = {}
+
```

Comment:
Let me try. I had issues with linting/type checking about mutable defaults in the past, but hopefully it is smart enough to realize that it is not a problem because the argument is annotated as an immutable `Mapping`...